Makefile

```diff
@@ -155,6 +155,13 @@ check-tests:
 	black --line-length 100 test_${PIPELINE_PACKAGE} --check
 	flake8 test_${PIPELINE_PACKAGE}
 
+## check-scripts:               run shellcheck
+.PHONY: check-scripts
+check-scripts:
+    # Fail if any of these files have warnings
+	chmod +x scripts/shellcheck.sh
```

Comment:
this script can be checked in as executable. ie, `chmod +x scripts/shellcheck.sh` and check it in, then this chmod can be removed